etna/lib/etna/client.rb

```diff
@@ -24,7 +24,7 @@ def route_path(route, params)
 
     def multipart_post(endpoint, content, &block)
       uri = request_uri(endpoint)
-      multipart = Net::HTTP::Post::Multipart.new uri.path, content
+      multipart = Net::HTTP::Post::Multipart.new uri.request_uri, content
```

Comment:
Necessary to keep, not drop, any query params that exist on the POST (which happens for upload_blob)